examples/usecases/incremental-training-with-layer-freezing.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>I think we should change the headers to fit more the different objectives.</p><p> </p><p>Building a Tow Tower Model</p><p>- Iteration 1: Using different Learning Rates</p><p>-- Training on First Day's Data</p><p>-- Training on Second Day's Data</p><p>- Iteration 2: Training with Freezing Layers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>